### DISCUSSION TITLE
Salesforce Release Notes Update

### DISCUSSION THREAD

**Discussion by AnanyaJha** [forcedotcom/salesforcedx-vscode#4319] [Ideas]
This Discussion introduces the **Salesforce Release Notes** pop up at the end of every major iteration for our IDE Experience team - in other words, roughly every 8 weeks. The purpose of these Release Notes is to improve _discoverability_ of new features within the IDE. 
Let us know if these Release Notes would be helpful for y'all and if there's specific content or a specific format you'd like to see!

### Discoverability
1. The Salesforce Extensions' Release Notes should appear for users when they launch a new major release version of the Salesforce Extensions within VS Code.

**New Setting:** Default to _True_. Setting to control whether or not to show Release Notes to user after a major release version update of the Salesforce Extensions. This should be modeled after the existing setting for the VS Code Release Notes.

<img width="573" alt="Screen Shot 2022-07-18 at 3 50 48 PM" src="https://user-images.githubusercontent.com/21326913/180889392-a0a7c714-9579-430f-b115-6788b63343c4.png">

2. The Salesforce Extensions' Release Notes should appear for users whenever they run the `SFDX: What's New` command in the Command Palette. This should be modeled after the existing `VS Code: What's New` command for the VS Code Release Notes.

### Content
The Release Notes page will contain the following sections:
- should contain a section for new **Feature Additions**
    - should allow user to click into the feature and _try it out_ directly from the Release Notes
- should contain a section for **Featured Fixes** from the iteration
- should contain a section highlighting **Contributions** from external users
- should contain links to Github repository

### Design Wireframes
Here are mockups designed by our User Experience team (thank you Meg!) to capture the current plans for the Release Notes:

![title](https://user-images.githubusercontent.com/21326913/180889923-86fd6ffa-105e-4cda-94a9-b08e9874381c.png)

![tryitout](https://user-images.githubusercontent.com/21326913/180889950-672486ff-39e6-4593-88c1-e1d96325a9e3.png)

![featuredfixes](https://user-images.githubusercontent.com/21326913/180889959-25d93131-4aa8-4e7f-994c-be68b01b3d49.png)

Related Discussion Post: https://github.com/forcedotcom/salesforcedx-vscode/discussions/3937

**Comment by jefersonchaves** [comment]
I like the idea. It is similar to other plugins and even how VsCode displays its Release Notes.
